### ISSUE TITLE
Problème de connexion

### ISSUE THREAD

**Issue by LeChatonTarnais** [Rolisteam/rolisteam#104]
Bonjour, 

Je ne comprend pas comment utiliser les serveurs, j'ai passé l'adresse IP du serveur et le port à un joueur et il n'arrive pas à se connecter, pourriez vous m'expliquer ?

Merci d'avance !

**Comment by LeChatonTarnais** [comment]
Ca me met Erreur de transmission : Le socket n'est pas connecté

**Comment by obiwankennedy** [comment]
Bonjour, 

Vous êtes probablement derrière un routeur/box de votre fournisseur internet ? (livebox/freebox/... si vous etes en France). 
1/ Vous confirmez cela ?

Aidez-vous [Ce test](http://www.rolisteam.org/php/test_ip.php) pour simuler la tentative de connection d'un joueur sur votre serveur. Ainsi, vous serez autonome pour tester.

2/ Si vous allez sur [ce site](http://www.mon-ip.com/), que voyait vous en rouge ?

Pour vous expliquer il y a 2 possibilités pour ce problème.

Soit votre réseau est en ipv6 et donc rolisteam vous donne une mauvaise adresse (car rolisteam donne de l'ip v4). Pour savoir si nous sommes dans ce cas, la réponse à la question 2 est primordiale.

Si vous etes en ipv4, c'est que votre routeur/machinbox n'est pas configurer pour laisser rentrer les connexions  de vos joueurs. Il va falloir configurer votre routeur pour lui dire d'accepter les connexions rentrantes et de les envoyer à votre ordinateur (et uniquement le votre, pas le telephone portable en wifi, ni la tablette etc...) 
Pour savoir, si nous sommes dans ce cas, le texte en rouge sur le site de la question 2 sera identique à l'adresse que vous a communiqué rolisteam. Il me faudra connâitre votre fournisseur d'acces pour vous envoyer vers le bon tutorial de configuration.

**Comment by LuCiuS00** [comment]
Bonjour je rencontre actuellement un problème avec rolisteam , quand je copie l'adresse de jeux j'obtient ça : "  Feature - Adresse IP du serveur:<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://rolisteam.org/ip.php">here</a>.</p>
<hr>
<address>Apache/2.4.29 (Ubuntu) Server at www.rolisteam.org Port 80</address>
</body></html>"

pourriez vous m'aider ?

**Comment by LuCiuS00** [comment]
normalement c'est une adresse énorme et ca ne marche pas du tout

**Comment by obiwankennedy** [comment]
Oui, l'adresse est donnée par le site web de rolisteam. Comme le site web est en train d'être déplacer il y a des petites erreurs. 
Pas de panique, pour l'adresse ip: il suffit d'aller: sur un site comme ça https://whatismyipaddress.com/fr/mon-ip

**Comment by LuCiuS00** [comment]
daccord merci
